src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -9,22 +9,25 @@
 #
```

Comment:
@BruceForstall, can we delete `run-pmi-diffs.py` and other unused scripts in this directory? More than one scripts in this directory have stale references: 4-6 years old docker containers, Jenkins CI which is unused since 2019, downloading emulators from 2016 blobs which return 404-Not-Found etc.

Reply:
Makes sense. Yes, we should do that. (But not in this change)